### ISSUE TITLE
Publish this sim with zoom/magnification

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#162]
From today's a11y planning meeting, we decided, that since this sim is being republished anyways, it could be nice to add zoom/mag as a feature to this as well. Tagging @jessegreenberg and @emily-phet so they are made aware of this issue.

@jessegreenberg what needs to be done to make this happen?

**Comment by zepumph** [comment]
There is a bit of momentum in this repo, so it would be good to ping @jessegreenberg again on the steps needed for this issue.

**Comment by zepumph** [comment]
After talking with @jessegreenberg during the a11y planning meeting today, I think we should just turn this on, and see how it goes!

**Comment by zepumph** [comment]
On hold until https://github.com/phetsims/chipper/issues/956

**Comment by zepumph** [comment]
@jessegreenberg, adding this seemed mostly normal and fine, but you have more experience with this so I thought I would pass it over to you for a spot check. One spot that seemed weird to me is trying to use the sliders when you zoom in a bit. The pan takes priority with much more strength than I would have thought. Is that intended behavior? You basically have to zoom back out again to use the slider. I see MAL doesn't have a slider anymore, so I wonder how much slider-time the pan/zoom has had during development and testing.

**Comment by jessegreenberg** [comment]
The result of the comments above was https://github.com/phetsims/scenery/issues/1057 which improved MultiListener and addressed these issues. That issue was reviewed and closed so this one can be as well.

**Comment by zepumph** [comment]
Yes, but note that the sim has not yet been published with zoom mag to my knowledge. Perhaps this is an outdated issue, because all sims from master will now get zoom/mag. The only reason to reopen would be if OL was a priority to republish, which it seems not to be.

**Comment by jessegreenberg** [comment]
Ah, yes you are right. I think OK to close this, the feature is working well but republishing this sim is no longer in the list of team goals.
